documentation/src/docs/asciidoc/user-guide/writing-tests.adoc

```diff
@@ -1037,6 +1037,8 @@ include::{testDir}/example/ParameterizedTestDemo.java[tags=implicit_conversion_e
 | `double`/`Double`          | `"1.0"`                                  -> `1.0d`
 | `Enum` subclass            | `"SECONDS"`                              -> `TimeUnit.SECONDS`
 | `java.io.File`             | `"/path/to/file"`                        -> `new File("/path/to/file")`
+| `java.lang.Class`          | `"java.lang.Integer"`                    -> `Class.forName("java.lang.Integer")` +
```

Comment:
This is now technically incorrect since we no longer use `Class.forName(String)` in the implementation.

What we have now is more powerful since it supports primitive types, arrays, etc.

So perhaps we should link to a footnote or callout that explains what is actually supported.

Reply:
Good point. Will polish after merging to `master`.

Reply:
Thanks